Gemfile

```diff
@@ -94,6 +94,7 @@ group :development do
 
   gem 'mina'
   gem 'mina-multistage', require: false
+  gem 'bullet'
```

Comment:
> I also added the bullet gem to identify possible problems in our queries, although I didn't find quite useful, maybe others can use it better.

I think exclude that from this PR if you didn't find it useful. I've only heard about it, but never actually used it.